dref/serializers.py

```diff
@@ -373,18 +374,21 @@ def update(self, instance, validated_data):
             dref_assessment_report = super().update(instance, validated_data)
             dref_assessment_report.needs_identified.clear()
             return dref_assessment_report
+
         # we don't send notification again to the already notified users:
         if 'users' in validated_data:
             to = {u.email for u in validated_data['users']
                   if u.email not in {t.email for t in instance.users.iterator()}}
         else:
             to = None
-        dref = super().update(instance, validated_data)
-        if to:
-            transaction.on_commit(
-                lambda: send_dref_email.delay(dref.id, list(to), 'Updated')
-            )
-        return dref
+        if modified_at and instance.modified_at and modified_at > instance.modified_at:
+            dref = super().update(instance, validated_data)
+            if to:
+                transaction.on_commit(
+                    lambda: send_dref_email.delay(dref.id, list(to), 'Updated')
+                )
+            return dref
```

Comment:
Throw error if
- modified_at is None, 'modified_at is required'
- modified_at != instance.modified_at, 'OBSOLETE_PAYLOAD'
Note: defined 'OBSOLETE_PAYLOAD' as a static value, client will use this string to identify this type of error. Please add a note in code for this as well.